packages/EasyDecision/src/Providers/ConfigMappingProvider.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Providers;
+
+final class ConfigMappingProvider extends AbstractMappingProvider
+{
+    /**
+     * @param string[] $decisionsConfig
+     */
+    public function __construct(array $decisionsConfig)
+    {
+        $this->setTypesMapping($decisionsConfig);
+    }
+
+    /**
+     * @param string[] $decisionsConfig
+     */
+    private function setTypesMapping(array $decisionsConfig): void
```

Comment:
I think that this method should be on the abstract mapping provider, this way regardless of where the mapping is coming from the abstract class provides the logic to validate the entries.

Reply:
Done.